hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/OzoneManagerServiceProviderImpl.java

```diff
@@ -228,7 +232,9 @@ public OMMetadataManager getOMMetadataManagerInstance() {
   @Override
   public void start() {
     LOG.info("Starting Ozone Manager Service Provider.");
-    scheduler = Executors.newScheduledThreadPool(1);
+    scheduler = Executors.newScheduledThreadPool(1,
+        new ThreadFactoryBuilder().setNameFormat(threadNamePrefix)
```

Comment:
Same comment as for SCM, let's use `threadNamePrefix + "SyncOM-%d"` or similar as name format.  Also, I guess we can store `threadNameFormat` as instance variable instead of the prefix to avoid duplication.